src/snowflake/cli/plugins/nativeapp/manager.py

```diff
@@ -47,6 +50,7 @@
     MissingPackageScriptError,
     UnexpectedOwnerError,
 )
+from snowflake.cli.plugins.nativeapp.feature_flags import FeatureFlag
```

Comment:
I know this is not the gist of the change but should we consider renaming `FeatureFlag` to `NativeAppFeatureFlags`? In this way we may avoid any ambiguity with existing `snowflake.cli.api.feature_flags.FeatureFlag.` WDYT?

Reply:
Isn't it common already to have the same files in multiple plugin directories if they define the same concepts? I'm thinking commands and manager, for example. Native apps code shouldn't rely on feature flags outside of its own IMHO, so at least now there shouldn't be ambiguity.